tests/cypress/integration/actions_users/issue_1810_login_logout.js

```diff
@@ -64,9 +78,40 @@ context('When clicking on the Logout button, get the user session closed.', () =
                 responce = await responce['headers']['set-cookie'];
                 const csrfToken = responce[0].match(/csrftoken=\w+/)[0].replace('csrftoken=', '');
                 const sessionId = responce[1].match(/sessionid=\w+/)[0].replace('sessionid=', '');
-                cy.visit(`/login-with-token/${sessionId}/${csrfToken}?next=/tasks/${taskId}`)
+                cy.visit(`/login-with-token/${sessionId}/${csrfToken}?next=/tasks/${taskId}`);
+                if (Cypress.browser.family !== 'chromium') {
+                    cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                        cy.contains('button', 'OK').click();
+                    });
+                }
                 cy.contains('.cvat-task-details-task-name', `${taskName}`).should('be.visible');
             });
         });
+
+        it('Incorrect user and correct password', () => {
+            cy.logout();
+            if (Cypress.browser.family !== 'chromium') {
+                cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                    cy.contains('button', 'OK').click();
+                });
+            }
+            login('randomUser123', Cypress.env('password'));
+            cy.url().should('include', '/auth/login');
+            cy.get('.cvat-notification-notice-login-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-login-failed');
+        });
+
+        it('Correct user and incorrect password', () => {
+            login(Cypress.env('user'), 'randomPassword123');
+            cy.url().should('include', '/auth/login');
+            cy.get('.cvat-notification-notice-login-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-login-failed');
+        });
+
+        it('Incorrect user and incorrect password', () => {
+            login('randomUser123', 'randomPassword123');
+            cy.url().should('include', '/auth/login');
+            cy.get('.cvat-notification-notice-login-failed').should('exist');
```

Comment:
Probably ``cy.closeNotification('.cvat-notification-notice-login-failed');`` is missed?

Reply:
Optional to close notice window in last step, but it be present.